src/write.c

```diff
@@ -481,7 +481,6 @@ static void avifEncoderBackupSettings(avifEncoder * encoder)
 static avifBool avifEncoderDetectChanges(const avifEncoder * encoder, avifEncoderChanges * encoderChanges)
 {
     const avifEncoder * lastEncoder = &encoder->data->lastEncoder;
-    *encoderChanges = 0;
```

Comment:
It may be better to leave this and document that the value will always be set meaningfully. This is a minor change could be its own PR to keep the review size down in this one.

Reply:
`avifEncoderDetectChanges()` follows a common convention that output parameters are set when the function succeeds. In this convention it is unspecified whether output parameters are set when the function fails, and the caller should only use the outputs after a successful call.

So the caller of `avifEncoderDetectChanges()` does not need to initialize the output before calling `avifEncoderDetectChanges()`.